<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestAppClass</class>
 <widget class="QMainWindow" name="QtTestAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>214</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>363</width>
    <height>214</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>363</width>
    <height>214</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtTestApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>59</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Check for Updates</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="checkingLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Checking updates...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qttestapp.qrc"/>
 </resources>
 <connections/>
</ui>
